src/lib.rs

```diff
@@ -144,29 +140,20 @@ impl LoggerUi {
             .stick_to_bottom(true)
             .show(ui, |ui| {
                 logs.iter().for_each(|(level, string)| {
-                    let string_format = format!("[{}]: {}", level, string);
-
-                    if !self.search_term.is_empty() && !self.match_string(string) {
+                    if (!self.search_term.is_empty() && !self.match_string(string))
+                        || !(self.loglevels[*level as usize - 1])
+                    {
                         return;
                     }
 
-                    if !(self.loglevels[*level as usize - 1]) {
-                        return;
-                    }
+                    let string_format = format!("[{}]: {}", level, string);
 
-                    match level {
-                        log::Level::Warn => {
-                            ui.colored_label(Color32::YELLOW, string_format);
-                        }
-                        log::Level::Error => {
-                            ui.colored_label(Color32::RED, string_format);
-                        }
-                        _ => {
-                            ui.label(string_format);
-                        }
-                    }
+                    let _ = match level {
```

Comment:
Is there a reason for the `let _`?

Reply:
You're totally right, this is purely a formatting thing, feel free to omit.